antarest/study/storage/variantstudy/model/command/create_link.py

```diff
@@ -57,11 +53,128 @@ def validate_series(
         return validate_matrix(v, new_values) if v is not None else v
 
     @model_validator(mode="after")
-    def validate_areas(self) -> "CreateLink":
+    def validate_areas(self) -> "AbstractLinkCommand":
         if self.area1 == self.area2:
             raise ValueError("Cannot create link on same node")
         return self
 
+    def to_dto(self) -> CommandDTO:
+        args = {
+            "area1": self.area1,
+            "area2": self.area2,
+            "parameters": self.parameters,
+        }
+        for attr in MATRIX_ATTRIBUTES:
+            if value := getattr(self, attr, None):
+                args[attr] = strip_matrix_protocol(value)
+        return CommandDTO(action=self.command_name.value, args=args, study_version=self.study_version)
+
+    def match(self, other: ICommand, equal: bool = False) -> bool:
+        if not isinstance(other, CreateLink):
```

Comment:
Should probably be replaced with `if not isinstance(other, self.__class__):` as you're inside AbstractCommand